clowdapp.yaml

```diff
@@ -80,19 +80,99 @@ objects:
           failureThreshold: 3
           httpGet:
             path: /
-            port: 8000
+            port: 9000
             scheme: HTTP
-          initialDelaySeconds: 30
-          periodSeconds: 10
+          initialDelaySeconds: 10
+          periodSeconds: 30
```

Comment:
Ditto, like to know why we increased this one?

Reply:
Same, sending request every 10 seconds just to determine weather pod is up seems to much hence changed it to 30.